osidb/filters.py

```diff
@@ -268,6 +271,16 @@ def is_major_incident_filter(self, queryset, name, value):
             ]
         )
 
+    def component_filter(self, queryset, name, value):
+        """
+        Based on the `value`, returns all Flaws which contains all components listed separated by colon.
+        This enables retro compactibility with deprecated field `component`.
+        """
+        if value:
+            components = [value.strip() for value in value[0].split(":")]
```

Comment:
I am not sure I understand this correctly but is the **colon** meant to be the separator of the component list in the query string? I think that comma is more common separator. Why colon?

Reply:
this function is used only on the old field `component`, if user try to filter with something like `/flaws?component=golang: net/http` it would convert the filter for the new field that stores `golang` and `net/http` as different components

Reply:
a second thought on that: since we changed to not parse the input of the user directly in the `component` (old) field, I believe I should just search as is, submitting the change.